modules/core/src/test/java/org/apache/ignite/internal/processors/metastorage/persistence/DmsDataWriterWorkerTest.java

```diff
@@ -346,7 +346,7 @@ private void write(String key, String val) throws IgniteCheckedException {
 
     /** */
     private DistributedMetaStorageHistoryItem histItem(String key, String val) throws IgniteCheckedException {
-        return new DistributedMetaStorageHistoryItem(key, JdkMarshaller.DEFAULT.marshal(val));
+        return new DistributedMetaStorageHistoryItem(key, new JdkMarshaller().marshal(val));
```

Comment:
change for TEST_JDK_MARSHALLER for consistency with other tests

Reply:
This class is not a subclass of GridCommonAbstractTest.